x-pack/plugins/security_solution/server/fleet_integration/fleet_integration.ts

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
this was both a rename (used to be called `ingest_integration` as well as a refactor. Prior version had all logic for the action that are taken in this module. Those have now been split up into separate modules (under `./handler/**`)

Reply:
are the handlers largely the same after the refactor?

Reply:
Yes - I just moved the code over. tests - which use the `fleet_integration` module - were unchanged.